src/Components/Cart.js

```diff
@@ -1,54 +1,75 @@
-import React, {Component} from 'react';
-import {connect} from 'react-redux';
-import {deleteCartItem} from '../redux/store';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import {
+  deleteCartItem,
+  addCartItemThunk,
+  createOrderThunk
+} from '../redux/store';
 
-class Cart extends Component{
-  render(){
-    const {cart, destroy, cartTotal} = this.props
-    if (cart.length === 0) {
+class Cart extends Component {
+  render() {
+    const { cart, destroy, orders, auth, products } = this.props;
+    const orderAuth = orders.filter(order => order.userId === auth.id);
```

Comment:
Is 'orderAuth just supposed to find a single order? If so, you can use .find instead of .filter and change 'orderAuth[0]' on line 15 to just be 'orderAuth'. This will still work, it's just a suggestion :)